app/controllers/application_controller.rb

```diff
@@ -39,6 +39,19 @@ def verify_admin
     (current_user.nil?) ? redirect_to(root_path) : (redirect_to(root_path) unless current_user.is_admin?)
   end
 
+  def tracker(ga_category, ga_action, ga_label, ga_value=nil)
```

Comment:
likewise here, the `=nil` is redundant as that's default if the param is not supplied anyway.